src/index.tsx

```diff
@@ -217,6 +230,22 @@ hither._registerSendMessage((msg) => {
 })
 
 const handleSetWorkspaceInfo = (workspaceInfo: WorkspaceInfo) => {
+  // important to send the subfeed watch messages before the reportClientInfo
+  // because otherwise our initial load will not include the recordings/sortings
+  apiConnection.sendMessage({
+    type: 'addSubfeedWatch',
```

Comment:
Here is where we make the subfeed watches for recordings/sortings in the workspace. Before this was done on the server side. Now it puts us in a position to implement workspace view in jupyter.